src/steps/update-signatures/update-signature.ts

```diff
@@ -43,7 +43,9 @@ export function updateSignature(file: string, data: Data): string {
       const argsNode = getBodyNode(path.node, 'Args');
 
       const isArgsKnown =
-        argsNode?.typeAnnotation.typeAnnotation.members.length > 0;
+        argsNode &&
+        argsNode.typeAnnotation.typeAnnotation.type === 'TSTypeLiteral' &&
```

Comment:
`argsNode.typeAnnotation.typeAnnotation.type` is not equal to `TSTypeLiteral` for the following code:

```ts 
export interface Signature {
  Args: unknown;
  Element: HTMLDivElement;
}
```